src/ts/createTyperighterPlugin.ts

```diff
@@ -37,6 +37,11 @@ export interface IPluginOptions<TMatch extends IMatch = IMatch> {
    * Is the plugin active as it starts?
    */
   isActive?: boolean;
+
+  /**
+   * The colours to use for document matches.
+   */
+  matchColours?: IMatchColours;
```

Comment:
We add this as a configuration value, so plugin consumers can provide their own colours. This will be important for us, as well as 3rd party users – we don't want to have to rerelease when our colour scheme changes!